students/nartowska_karina/lesson_06_dicts_tuples_sets_args_kwargs/args_sum.py

```diff
@@ -0,0 +1,14 @@
+def sum_all(*args):
+    sum = 0
```

Comment:
```sum``` is built-in python function so this name should be avoided (Please have a look at rest of built-in functions: https://docs.python.org/3/library/functions.html). 

This could be done as well using sum function, look at this:
```
def sum_all(*args):
      return sum(args)
```

Of course, your solution is ok